glusterd2/commands/volumes/volume-option.go

```diff
@@ -139,6 +169,8 @@ func registerVolOptionStepFuncs() {
 		{"vol-option.Validate", optionSetValidate},
 		{"vol-option.XlatorActionDoSet", xlatorActionDoSet},
 		{"vol-option.XlatorActionUndoSet", xlatorActionUndoSet},
+		{"vol-option.XlatorActionDoReSet", xlatorActionDoReSet},
```

Comment:
Please move registering these to `glusterd2/commands/volumes/volume-reset.go` file.

Reply:
done